tests/test_operations.rs

```diff
@@ -41,12 +41,12 @@ fn test_op_log() {
         &["op", "log", "--config-toml", "ui.relative-timestamps=true"],
     );
     insta::assert_snapshot!(&stdout, @r###"
-    @ 45108169c0f8 test-username@host.example.com 21 years ago - 21 years ago
+    @ 45108169c0f8 test-username@host.example.com 21 years ago
```

Comment:
What do you think about making the relative form be "21 years ago, for 34 milliseconds" (or however `timeago` renders millisecond durations)? That seems a little useful, but maybe it's a strange to have `ui.relative-timestamp` also make the difference relative?

Reply:
That's a good idea. It did make the third commit a bit more complicated, but should look very friendly to users.